packages/editor-ui/src/components/Sticky.vue

```diff
@@ -146,6 +147,18 @@ export default mixins(externalHooks, nodeBase, nodeHelpers, workflowHelpers).ext
 				this.$store.commit('setActiveNode', null);
 			}
 		},
+		onMarkdownLinkClick(event: Event) {
+			if(event.target && event.currentTarget) {
+				// Track clicks on sticky links
+				const clickedURL = (event.currentTarget as Element).getAttribute('href') || '';
+				// If it's a YT link and it contains a N8n quickstart thumbnail, it's a welcome video link
+				const isWelcomeVideo = clickedURL.startsWith('https://www.youtube.com/') &&
+				 ((event.target as Element).localName === 'img' && (event.target as Element).getAttribute('alt') === 'n8n quickstart video');
+
+				const type = isWelcomeVideo ? 'welcome_video' : clickedURL === '/templates' ? 'templates' : 'other';
```

Comment:
to avoid any false positives can we also check that the note name matches our unique onboarding name?
```suggestion
				const type = isOnboardingNote && isWelcomeVideo ? 'welcome_video' : isOnboardingNote && clickedURL === '/templates' ? 'templates' : 'other';
```

Reply:
Agree